plugins/autogems.cpp

```diff
@@ -281,14 +328,12 @@ IMPLEMENT_VMETHOD_INTERPOSE(autogem_hook, render);
 
 DFhackCExport command_result plugin_onstatechange(color_ostream &out, state_change_event event) {
     if (event == DFHack::SC_MAP_LOADED) {
-        if (enabled && World::isFortressMode()) {
-            // Determine whether auto gem cutting has been disabled for this fort.
-            auto config = World::GetPersistentData(CONFIG_KEY);
-            running = config.isValid() && !config.ival(0);
-            last_frame_count = world->frame_counter;
-        }
+        running = enabled && World::isFortressMode();
+        //if ( !World::isFortressMode() )
+        //    return CR_OK;
+        load_config(out);
```

Comment:
Did you intend to change the behavior here?  Is this still safe in arena mode?  Did you mean to have the plugin up and enable itself if it had been active when the newly loaded fortress was last saved?